ui/src/views/network/IpAddressesTab.vue

```diff
@@ -302,6 +300,26 @@ export default {
     }
   },
   inject: ['parentFetchData'],
+  computed: {
+    listApiParams () {
+      const params = {
+        zoneid: this.resource.zoneid,
+        domainid: this.resource.domainid,
+        account: this.resource.account,
+        forvirtualnetwork: true,
+        allocatedonly: false
+      }
+      if (['nsx', 'netris'].includes(this.zoneExtNetProvider?.toLowerCase())) {
+        params.forprovider = true
+      }
+      return params
+    },
+    listApiParamsForAssociate () {
+      const params = this.listApiParams
+      params.state = 'Free,Reserved'
```

Comment:
@shwstppr I tested the changes, it is not listing any IPs now

<img width="984" height="574" alt="Image" src="https://github.com/user-attachments/assets/1759df47-40b5-4a0e-97db-94a82f2c9f6b" />

Seems to be problem with these params

http://10.0.32.194:8080/client/api/?page=1&pagesize=20&zoneid=eb21bc9a-4c08-4456-938b-af605ef36aae&domainid=d3a3e350-a40d-11f0-b82c-1e009c000155&account=admin&forvirtualnetwork=true&allocatedonly=false&**state=Free,Reserved**&showicon=true&command=listPublicIpAddresses&response=json&sessionkey=fSa-6YW14skm_KeFEJvTnLPpQQs